src/constants/common.constants.tsx

```diff
@@ -1,51 +1,52 @@
 const Paths = {
   main: {
     path: '/',
-    title: 'Main',
+    title: 'Главная',
```

Comment:
Why RUSSIAN??? NO RUSSIAN! Just ENGLISH! ))

Reply:
The site is in Russian and the headlines are in Russian, respectively.)